docs/docs/webchat/webchat-connectivity.md

```diff
@@ -0,0 +1,54 @@
+---
+id: webchat-connectivity
+title: Connectivity
+---
+
+Sometimes, the connection can be lost between the server and the chatbot. To handle this situation, you can set specific parameters to detect the connection issue.
```

Comment:
"between the server and the **chatbot**" -> "between the server and the **webchat**"